src/resources/filters/crossref/meta.lua

```diff
@@ -38,7 +37,8 @@ function crossrefMetaInject()
           )
 
           inject(
-            "\\newcommand*\\listoflistings{\\listof{codelisting}{" .. listOfTitle("lol", "List of Listings") .. "}}\n"
+            "\\@ifundefined{listoflistings}{\\newcommand*\\listoflistings{\\listof{codelisting}{" ..
+            listOfTitle("lol", "List of Listings") .. "}}}{}\n"
```

Comment:
Same here. It may be more applicable as this is for Float and this will make the crossreference system for code listing to work. If `\listoflisting` is not doing `\\listof{codelisting}` as we defined it, then maybe this will be more a problem than trying to support minted package. 

So `\renewcommand` may be required to overwrite for crossref to work (and not opted-out)